in-depth/submission_guidelines.md

```diff
@@ -3,31 +3,31 @@ layout: docs
 title: Theme & Plugin Submission Guidelines
 headline: Theme & Plugin Submission Guidelines
 description: |
-  Submitting your Theme or Plugin for inclusion on the website is very easy.<br>
+  Submitting your theme or plugin for inclusion on the website is very easy.<br />
```

Comment:
Okay, well don't expect me to start any time soon. It was a hard enough habit to break. :wink: